### ISSUE TITLE
Numbers look larger with muted heading 

### ISSUE THREAD

**Issue by anthonydillon** [canonical/vanilla-framework#4697]
**Describe the bug**

Numbers and letters are different heights using the `p-muted-heading` pattern.

**To Reproduce**

1. Go to https://vanillaframework.io/docs/patterns/muted-heading
2. Update the content in the inspector

**Expected behavior**

I would expect to see the letters and numbers being the same height.

**Screenshots**

![image](https://user-images.githubusercontent.com/1413534/224300739-22e8a905-2fd6-4327-878e-cc244fad5e81.png)

**Comment by lyubomir-popov** [comment]
@bartaz @anthonydillon yes, the proper numbers that work well with small caps are called [text figures]( ![image](https://user-images.githubusercontent.com/2741678/224367273-debe5b39-0b84-40e0-a56f-a57e8ef6f9a9.png)) (as opposed to what we have now, which is called titling figures). We are waiting on the quote for text figures to be approved and from that point we'll have them in two weeks. Shouldn't be too long, we are aware and working on it.
